recipes/debugpy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "debugpy" %}
+{% set version = "1.0.0b9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: d57e4854c78d007d9516b8d3ae0c827cd6df87374d6d56d9c4fbc7adfa83546b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: |
+    pushd src/debugpy/_vendored/pydevd/pydevd_attach_to_process
+    rm *.so *.dll *.dylib *.exe *.pdb
+    cd linux_and_mac
+    # linux
+    ${CXX} -m64 -shared -o attach_linux_amd64.so -fPIC -nostartfiles attach.cpp  # [linux]
```

Comment:
Use CXXFLAGS and LDFLAGS instead of the manual flags here. Currently you are building in the extremely slow unoptimized mode (-O0)

Reply:
Using *FLAGS und SHLIB_EXT should allow you to have the same commands for OSX and Linux.